src/app/Library/CrudPanel/CrudFilter.php

```diff
@@ -285,7 +285,7 @@ public function label($value)
      */
     public function values($value)
     {
-        $this->values = $value;
+        $this->values = is_callable($value) ? $value() : $value;
```

Comment:
We had trouble using this kind of check in the past, who knows what values could endup there :smiley: 

 Just to make sure we don't get into the same hole I would go here with: 

`$this->values = (!is_string($value) && is_callable($value)) ? $value() : $value;`

If the string is a php core function we know what is going to happen :+1: